rocketmq-tools/rocketmq-admin/rocketmq-admin-tui/README.md

```
@@ -1,13 +1,28 @@
+
+## How to Run
 
 ```shell
-cargo run -p rocketmq-tui
+cargo run -p rocketmq-admin-tui
 ```
 
-### design
+## Design
 
 ![](../../../resources/rocketmq-cli-ui.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)